frappe/core/doctype/installed_application_details/installed_application_details.json

```diff
@@ -0,0 +1,49 @@
+{
+ "actions": [],
+ "creation": "2020-05-11 17:44:54.674657",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "application_name",
+  "application_version",
+  "git_branch"
+ ],
+ "fields": [
+  {
+   "fieldname": "application_name",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Application Name",
+   "read_only": 1,
+   "reqd": 1
+  },
+  {
+   "fieldname": "application_version",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Application Version",
+   "read_only": 1,
+   "reqd": 1
+  },
+  {
+   "fieldname": "git_branch",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Git Branch",
+   "read_only": 1,
+   "reqd": 1
+  }
+ ],
+ "istable": 1,
+ "links": [],
+ "modified": "2020-05-11 19:54:07.141641",
+ "modified_by": "Administrator",
+ "module": "Core",
+ "name": "Installed Application Details",
```

Comment:
DocType name should be singular unless it's a Single DocType.

ref: https://github.com/frappe/erpnext/wiki/Naming-Guidelines#doctype-naming